bin/i18n/resources/apps/external_sources.rb

```diff
@@ -14,6 +14,12 @@ def self.sync_in
 
           FileUtils.mkdir_p(I18N_SOURCE_DIR_PATH)
 
+          # Preparing Oceans tutorial
+          # TODO: move the file to the `external-sources` folder instead of `blockly-mooc`
+          labs_dir = CDO.dir(File.join(I18N_SOURCE_DIR, 'blockly-mooc'))
```

Comment:
I know it is weird to have content going to `blockly-mooc` (i.e. Apps) in the `external_sources` module, but for now we will keep it this way.